File: Loops/fizzbuzz/fizzbuzz.rb
fizzbuzzes = (1..100).map do |i|
  if i % 15 == 0 then "fizzbuzz"
  elsif i % 3 == 0 then "fizz"
  elsif i % 5 == 0 then "buzz"
  else i
  end
end

def fizzbuzz(num)
  if num.is_a?(Integer) == false
    raise TypeError, "Please pass in an integer."
  end

  if num % 15 == 0 then "fizzbuzz"
  elsif num % 3 == 0 then "fizz"
  elsif num % 5 == 0 then "buzz"
  else num
  end
end

File: todo41/src/fizzBuzz.js
function fizzBuzz() {
  for (var i = 0; i < 101; i++) {
    if (i % 15 == 0) {
      return "fizzbuzz";
    }
    else if (i % 5 == 0) {
      return "buzz";
    }
    else if (i % 3 == 0) {
      return "fizz";
    }
    else {
      return i;
    }
  }
}